.github/scripts/tkn_push_bundles.sh

```diff
@@ -14,18 +14,24 @@ printf 'bundle_dirs: %s\n' "${!bundle_dirs[@]}"
 
 # Main loop
 printf 'Pushing any new/modified bundle definitions.\n'
-for BUNDLE_DIR in "${!bundle_dirs[@]}"
+for VERSION_DIR in "${!bundle_dirs[@]}"
 do
-  if [[ ! -d "$BUNDLE_DIR" ]]
+  if [[ ! -d "$VERSION_DIR" ]]
   then
-    printf 'Skipping non-existing bundle-path: %s\n' "$BUNDLE_DIR"
+    printf 'Skipping non-existing bundle-path: %s\n' "$VERSION_DIR"
     continue
   fi
   BUNDLE_FILES=()
+  BUNDLE_DIR=${VERSION_DIR%/*}
   BUNDLE_NAME=${BUNDLE_DIR##*/}
-  printf '* Bundle_dir: %s\n' $BUNDLE_DIR
+  RESOURCE_DIR=${BUNDLE_DIR%/*}
+  RESOURCE_TYPE=${RESOURCE_DIR#*/}
+  printf '* Version dir: %s\n' $VERSION_DIR
```

Comment:
all these statements seem a little excessive to me. If we want to print so much data, can we just say like "processing yaml file a/b/c/d/X.yaml" or something and cut down on these variables? It is hard to follow imo

Reply:
Agreed. I've refactored those shell variables into positional-parameters.
I'm not sure it's any easier to read... I had to add a comment explaining what the positional parameters represent.
But the code is certainly more simplified from a runtime perspective, avoiding repetitive string parsing on each successive part of the YAML file path.

Reply:
I like it a lot better now and think the comment you added is very useful